packages/front-end/components/Share/Presentation.tsx

```diff
@@ -73,53 +74,41 @@ const Presentation = ({
   ).forEach((eid) => {
     // get the results in the right shape:
     const e = em.get(eid);
-
     // get the info on which variation to mark as winner/loser
-    const variationExtra = [];
+    const variationExtra: JSX.Element[] = [];
     let sideExtra = <></>;
     const variationsPlural =
-      // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-      e.experiment.variations.length > 2 ? "variations" : "variation";
+      (e?.experiment?.variations?.length || 0) > 2 ? "variations" : "variation";
```

Comment:
minor copy note ('0 variations' vs '0 variation')
```suggestion
      (e?.experiment?.variations?.length || 0) !== 1 ? "variations" : "variation";
```